docs/guides/essentials/overview.md

```diff
@@ -1,2 +1,94 @@
 # Overview
 
+Daffodil consists of a few different layers, each which are appropriate for different use cases.
+
+## Driver
+The driver layer is the most powerful layer. It interacts with external platforms and structures data into Daffodil's specific model format. This enables a single frontend application to interface with multiple platforms without dealing with the differences or intricacies of each of those platforms.
+
+Driver usage is straightforward; simply call a method on the driver class and use the result! Nearly all driver methods return an [`Observable`](https://rxjs.dev/api/index/class/Observable) to encapsulate the async behavior of network requests.
+
+```ts
```

Comment:
can you throw these examples into a component.ts file and lint them? the tabs for indent causes the examples to render with extra spacing vs two spaces.